props/saber_caiwyn_buttons.h

```diff
@@ -0,0 +1,604 @@
+// Caiwyn's Lightsaber Controls
+// This config is designed to provide fast, low-latency response to button presses for the most basic functions (blaster blocks, lockups, and clashes) by omitting, or limiting non-critical features.
+// At least 2 buttons are required.  A third button can be added, but is not necessary.
+//
+// While saber is DEACTIVATED:
+// Start/Stop Track: Click Aux
+// Track Player Mode: Double-click Aux
+//   This cycles through three playback modes:
+//   1. Play a single track and stop
+//   2. Repeat a single track in a loop
+//   3. Repeat all tracks in a loop
+//   Tracks must be in <font>/tracks/*.wav, and will be selected in alphabetical order.
+// Next Track: Hold Aux while track is playing
+// Next Preset: Hold Aux when track is not playing
+// Check Battery: Double-click and hold Aux
+// Activate Saber: Press Power
+// Activate Saber and Start Track: Hold Aux or Aux2 and press Power
+//
+// While saber is ACTIVATED:
+// Non-impact Clash/Lockup: Click Power (This generates a clash/lockup effect with no impact to the blade; quick press for a short clash, hold for a lockup)
+// Blaster Block: Click Aux
+// Lightning Block: Hold Aux
+// Lockup: Hold Aux during impact
+// Drag: Hold Aux during impact with saber pointed down.
+// Melt: Hold Aux and stab
+// Enter Volume Menu: Double-click and hold Aux (Be aware that the first click will trigger a blaster block)
+//   Increase Volume: Rotate hilt right while in Volume Menu
+//   Decrease Volume: Rotate hilt left while in Volume Menu
+//   Save Changes and Exit Volume Menu: Click Power
+//   Discard Changes and Exit Volume Menu: Click Aux
+// Enter Color Change Mode (Color Wheel): Triple-click and hold Aux (Be aware that the first two clicks will trigger blaster blocks)
+//   Change Color: Rotate hilt while in Color Change Mode
+//   Color Zoom: Press & Hold Power and Rotate hilt while in Color Change Mode
+//   Save Changes and Exit Color Change Mode: Release Power
+//   Reset Color and Exit Color Change mode: Click Aux
+// Deactivate Saber: Hold Aux or Aux2 and press Power
+//
+// You will need the following sound files in order for menus to work properly:
```

Comment:
Are all of these sounds available in the voice packs?
If so, maybe add an url to where they can be found...

Reply:
Most of them are; I used standard sounds that were used by Fett263's button config file whenever possible.  I have specified separate vmend and ccend files that are Selected depending on whether the user saves or cancels a volume or color change, but if you only have one, the code will use the same file for both.  The only sound that isn't already available is monce.wav.  The rest are all standard.  I'm putting together my own voice pack that will include the extra voice prompts I use, but it's not quite finished.

Reply:
I am still working on the mnum*.wav files. The config works without them; and any missing file is replaced by a beep code or a talkie.Say(). The standard edit mode menu wav files will work in a pinch.

Reply:
Don't explain it to me though.. It would be much better if you explain it in the file so that everybody who reads the file will know which files are expected and where to find the files needed.

Reply:
Finished the voice pack and made it publicly available.  There is now a link in the instructions.